### PR TITLE
Music assistant blog

### PR DESCRIPTION
**Pull request by googanhiem**
## Proposed change

Music Assistant 2 release blog

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2024-05-09-music-assistant-2.markdown [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+date: 2024-05-09 00:00:01
+date_formatted: "May 09, 2024"
+author: Marcel van der Veldt
+comments: true
+categories: Music-Assistant
+og_image: /images/blog/2024-05-music-assistant/art.jpg
+---
+<p><img src='/images/blog/2024-05-music-assistant/art.jpg' class='no-shadow' /></p>
+
+Today, exactly five years ago, I started working on [Music Assistant](https://music-assistant.io/). What began as a quick script to sync my playlists when I wanted to switch between streaming providers grew into a beast on its own.
```
Comment (balloob) [inline source/_posts/2024-05-09-music-assistant-2.markdown]: Should we mention his name? 
```suggestion
Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script to sync my playlists when I wanted to switch between streaming providers grew into a beast on its own.
```

Comment (balloob) [reply source/_posts/2024-05-09-music-assistant-2.markdown]: Also, should there be a comma in second sentence?

Comment (balloob) [reply source/_posts/2024-05-09-music-assistant-2.markdown]:
```suggestion
Today, exactly five years ago, I started working on [Music Assistant](https://music-assistant.io/). What began as a quick script, sync my playlists so I could switch between streaming providers, grew into a beast on its own.
```

File: source/_posts/2024-05-09-music-assistant-2.markdown [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+What I’d like to call a “music library manager” that gives you full control of playing YOUR music on your players. Just like Home Assistant centralizes all your devices and services to work together, Music Assistant does the same with your music sources and smart audio-playing devices. This is just one of the many [Home Assistant Add-on’s](https://www.home-assistant.io/addons/) that can massively expand its smart capabilities.
```
Comment (balloob) [inline source/_posts/2024-05-09-music-assistant-2.markdown]: Call what? This sentence feels out of the blue as a first sentence.

Comment (balloob) [reply source/_posts/2024-05-09-music-assistant-2.markdown]:
```suggestion
Music Assistant is what I’d like to call a “music library manager” – it gives you full control of playing YOUR music on your players. Just like Home Assistant centralizes all your devices and services to work together, Music Assistant does the same with your music sources and smart audio-playing devices. This is just one of the many [Home Assistant Add-on’s](https://www.home-assistant.io/addons/) that can massively expand its smart capabilities.
```

File: source/_posts/2024-05-09-music-assistant-2.markdown [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+date: 2024-05-09 00:00:01
+date_formatted: "May 09, 2024"
+author: Marcel van der Veldt
+comments: true
+categories: Music-Assistant
+og_image: /images/blog/2024-05-music-assistant/art.jpg
+---
+<p><img src='/images/blog/2024-05-music-assistant/art.jpg' class='no-shadow' /></p>
+
+Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script to sync my playlists when I wanted to switch between streaming providers grew into a beast on its own.
```
Comment (balloob) [inline source/_posts/2024-05-09-music-assistant-2.markdown]:
```suggestion
Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script, to sync my playlists so I could switch between streaming providers, grew into a beast on its own.
```

Comment (googanhiem) [reply source/_posts/2024-05-09-music-assistant-2.markdown]: Commas work, but it still needs the "to" before "sync"

File: source/_posts/2024-05-09-music-assistant-2.markdown [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+categories: Music-Assistant
+og_image: /images/blog/2024-05-music-assistant/art.jpg
+---
+<p><img src='/images/blog/2024-05-music-assistant/art.jpg' class='no-shadow' /></p>
+
+Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script, to sync my playlists so I could switch between streaming providers, grew into a beast on its own.
+
+Music Assistant is what I’d like to call a “music library manager” – it gives you full control of playing YOUR music on your players. Just like Home Assistant centralizes all your devices and services to work together, Music Assistant does the same with your music sources and smart audio-playing devices. This is just one of the many [Home Assistant Add-on’s](https://www.home-assistant.io/addons/) that can massively expand its smart capabilities.
+
+<p class="img"><img src='/images/blog/2024-05-music-assistant/how-it-works.jpg'/>From source to Music Assistant to speaker</p>
```
Comment (balloob) [inline source/_posts/2024-05-09-music-assistant-2.markdown:22]:
```suggestion
{% my supervisor_addon badge addon="d5369777_music_assistant_beta" repository_url="https://github.com/music-assistant/home-assistant-addon" %}

<p class="img"><img src='/images/blog/2024-05-music-assistant/how-it-works.jpg'/>From source to Music Assistant to speaker</p>
```

File: source/_posts/2024-05-09-music-assistant-2.markdown [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+categories: Music-Assistant
+og_image: /images/blog/2024-05-music-assistant/art.jpg
+---
+<p><img src='/images/blog/2024-05-music-assistant/art.jpg' class='no-shadow' /></p>
+
+Today, exactly five years ago, I, Marcel, started working on [Music Assistant](https://music-assistant.io/). What began as a quick script, to sync my playlists so I could switch between streaming providers, grew into a beast on its own.
+
+Music Assistant is what I’d like to call a “music library manager” – it gives you full control of playing YOUR music on your players. Just like Home Assistant centralizes all your devices and services to work together, Music Assistant does the same with your music sources and smart audio-playing devices. This is just one of the many [Home Assistant Add-on’s](https://www.home-assistant.io/addons/) that can massively expand its smart capabilities.
+
+{% my supervisor_addon badge addon="d5369777_music_assistant_beta" repository_url="https://github.com/music-assistant/home-assistant-addon" %}
```
Comment (balloob) [inline source/_posts/2024-05-09-music-assistant-2.markdown]: Update this to stable when Marcel releases it.

Comment (balloob) [inline source/_posts/2024-05-09-music-assistant-2.markdown]: The pitch about add-ons feels weird. Better put below the button.

```suggestion
Music Assistant is what I’d like to call a “music library manager” – it gives you full control of playing YOUR music on your players. Just like Home Assistant centralizes all your devices and services to work together, Music Assistant does the same with your music sources and smart audio-playing devices.

{% my supervisor_addon badge addon="d5369777_music_assistant_beta" repository_url="https://github.com/music-assistant/home-assistant-addon" %}

_Music Assistant is distributed as a [Home Assistant add-on](https://www.home-assistant.io/addons/) – our way to easily extend your Home Assistant installation with third party applications. If you use Home Assistant Container, follow the [MA installation instructions](https://music-assistant.io/installation/#tertiary-installation-method-docker-image)._
```

Comment (balloob) [reply source/_posts/2024-05-09-music-assistant-2.markdown]:
```suggestion
{% my supervisor_addon badge addon="d5369777_music_assistant" repository_url="https://github.com/music-assistant/home-assistant-addon" %}
```

Comment (marcelveldt) [reply source/_posts/2024-05-09-music-assistant-2.markdown]: addon is placed on the repo - that first part stays the same right (the slug) ?

Comment (marcelveldt) [reply source/_posts/2024-05-09-music-assistant-2.markdown]: https://github.com/music-assistant/home-assistant-addon/tree/main/music_assistant

**Comment by balloob** [comment]
Done in other pr.

**Pull request closed without merging**